### DISCUSSION TITLE
Please tell me your hobby

### DISCUSSION THREAD

**Discussion by watagit** [NAIST-SE/AttractingContributors#578] [General]
I like coffee, trip and hot spring.
Please tell me your hobby!

**Comment by olehvgrn** [comment]
Cooking is my passion

**Comment by CodyBra** [comment]
I love kayaking with friends and playing musical instruments. Just imagine how cool you can spend time on the weekend! Pack your things and go to some island, take a ukulele with you, make a fire in the evening and sing your favorite songs while frying marshmallows! I love this pastime!
